resources/overrides/jquery.sap.mobile.d.ts

```diff
@@ -11,7 +11,3 @@ interface JQueryStatic {
 	 */
 	device: object;
```

Comment:
`JQueryStatic` from this file would be extended by the one in `resources/overrides/jquery.sap.d.ts`.

I was wondering wether we need to keep this one or shall we just merge them explicitly

Reply:
Do you mean having them in one file? This is on purpose to allow easy detection (see `isSymbolOfJquerySapType` in `FileLinter.ts`)

Reply:
To be honest, I don't mind it. We just need to be aware of the side effects when merging TS definitions. So, maybe just mentioning as a comment that this interface has another definition in `resources/overrides/jquery.sap.d.ts` and they could (eventually) be merged, so we would be aware of the eventual side effects.
This is the TS feature I'm reffering to: https://www.typescriptlang.org/docs/handbook/declaration-merging.html